### ISSUE TITLE
App is not available in my country on play store.

### ISSUE THREAD

**Issue by dejanmilivojevic** [moneymanagerex/android-money-manager-ex#492]
Hi,

I'm unable to download app from google play store. I get "This item isn't available in your country" and since it is a open source app it doesn't make sense not to be available everywhere. Can you please fix this.

**Comment by MisterY** [comment]
Hi, Dejan,

Thanks for the tip! I've just checked the distribution list. It seems that Serbia was under sanctions at a time when the app was originally published as only the sanctioned countries are excluded and can not be added back. Serbia was the only country available to include so the above would be the only explanation. 
You are absolutely right. As an Open Source app, we certainly don't exclude people from downloading. On the contrary, we have recently published the app on Amazon store. 
All the releases are always available from GitHub, too.
Hope we see more feedback from Serbia after the inclusion! A translation into Serbian would also be great!
